### PR TITLE
Fix tuning sliders

### LINKED ISSUE #2181: Gyro Filter Slider stays active when all lowpass turned off.
I have noticed when i turn off the gyro filter completely that the slider stays active and shows 1.05. All the buttons are clicked to the off position from the slider being originally at 2x. Also in the lua script it shows a min value of 0 and a max value of 1000. Im not sure how to check to see if the filter is dynamically jumping from 0 to 1000 or if it is truly off.

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes #2181

### CODE REVIEW

**Review by McGiverGim — APPROVED** [review]
Thanks for looking at this!

File: src/js/TuningSliders.js [hunk 1]
```diff
@@ -208,24 +208,24 @@ TuningSliders.updateFilterSlidersDisplay = function() {
...
+        $('.tuningFilterSliders .sliderLabels tr:nth-child(2)').show()
         this.cachedGyroSliderValues = true;
     }
 
-    if ($('.pid_filter input[name="dtermLowpassDynMinFrequency"]').val() != Math.round(this.FILTER_DEFAULT.dterm_lowpass_dyn_min_hz * this.dtermFilterSliderValue) ||
-        $('.pid_filter input[name="dtermLowpassDynMaxFrequency"]').val() != Math.round(this.FILTER_DEFAULT.dterm_lowpass_dyn_max_hz * this.dtermFilterSliderValue) ||
-        $('.pid_filter select[name="dtermLowpassDynType"]').val() != this.FILTER_DEFAULT.dterm_lowpass_type ||
-        $('.pid_filter input[name="dtermLowpass2Frequency"]').val() != Math.round(this.FILTER_DEFAULT.dterm_lowpass2_hz * this.dtermFilterSliderValue) ||
-        $('.pid_filter select[name="dtermLowpass2Type"]').val() != this.FILTER_DEFAULT.dterm_lowpass2_type) {
+    if (parseInt($('.pid_filter input[name="dtermLowpassDynMinFrequency"]').val(), 10) !== Math.round(this.FILTER_DEFAULT.dterm_lowpass_dyn_min_hz * this.dtermFilterSliderValue) ||
```
Comment (mikeller) [inline src/js/TuningSliders.js:224]: Lots of gremlins in the configurator based slider code...

Thanks for fixing this, but please keep in mind that once the firmware 'sliders' functionality (https://github.com/betaflight/betaflight-configurator/pull/1821) has been implemented, a lot of this code will probably go away or have to be changed, so there isn't necessarily a lot of value into spending more time in fixing it right now.

**Review by mikeller — APPROVED** [review]

File: src/js/TuningSliders.js (hunk 1 above)
Comment (McGiverGim) [reply src/js/TuningSliders.js:224]: I think this has been abandoned, I dunt know if we will get it some time

Comment (McGiverGim) [reply src/js/TuningSliders.js:224]: Oh, I've seen the firmware part is now under development again. I was not aware of that... 🤗

Comment (mikeller) [reply src/js/TuningSliders.js:224]: According to @IvoFPV, the firmware tuning sliders / groups are almost ready - it's just that they don't fit into flash on STM32F7X2 at the moment.

**Pull request merged**